mkdocs.yml

```
@@ -31,7 +31,9 @@ extra:
     - icon: fontawesome/solid/leaf
       link: https://orchardcore.net
     - icon: fontawesome/brands/github-alt
-      link: https://github.com/OrchardCMS/OrchardCore.Commerce
+      link: https://github.com/OrchardCMS/OrchardCore.Commerce    
```

Unnecessary trailing whitespace.
```suggestion
      link: https://github.com/OrchardCMS/OrchardCore.Commerce
```